packages/appstore/src/components/wallet-transfer/components/transfer-account-selector/transfer-account-selector.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { useFormikContext } from 'formik';
+import { Div100vhContainer, Icon, Modal, Text, ThemedScrollbars } from '@deriv/components';
+import { useTransferBetweenAccounts } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import TransferAccountList from './transfer-account-list';
+import { WalletTransferTile } from '../wallet-transfer-tile';
+import { WalletJurisdictionBadge } from 'Components/wallet-jurisdiction-badge';
+import { getAccountName } from 'Constants/utils';
+import type { TTransferAccount } from 'Types';
+import type { TInitialValues } from '../../types';
+import './transfer-account-selector.scss';
+
+type TTransferAccountSelectorProps = {
+    contentScrollHandler?: React.UIEventHandler<HTMLDivElement>;
+    is_wallet_name_visible?: boolean;
+    name: 'from_account' | 'to_account';
```

Comment:
```suggestion
    type: 'from_account' | 'to_account';
```
Can we rename this to `type`?